tor/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '3.4.0'
+__version__ = '3.5.0'
```

Comment:
Shouldn't this be 3.4.1? Semvar rules, and everything?

Reply:
Look at what it's being merged into ;) While yes, normally I would actually leave this change undone, we're switching to feature branches -- that's why your PR was moved against the 3.5.0 release branch :)

Reply:
But since you brought it up, I'll go ahead and roll that back.

Reply:
Oh—I didn't see that.

yeah, probably best to have that in the final PR.